src/View/User/Includes/_course_tab.html.twig

```diff
@@ -0,0 +1,82 @@
+        </p>
+    </div>
+    <div class="tab_title_text">
+    <p class="tab_title">
+        INSCRIPTION
+    </p>
+    <p class="tab_text">
+        </br> Lorem ipsum dolor sit amet. Sed doloribus id eligendi rerum eos impedit dolore eos excepturi quisquam et corporis Quis At sunt voluptatem ex ullam dolorum. Et corrupti tempore id voluptatibus atque eos itaque enim non impedit laudantium est excepturi sunt et quia saepe! Non accusantium quia id culpa iure sed fugiat optio a reiciendis ipsum eum sunt nostrum. 
+    </p>
+    </div>
+    <div class="btn_insc">
+        <button class="btn_inscription" type="button">
+            JE M'INSCRIS !
+        </button>
+    </div>
+    <p class="tab_title">
+        DETAILS
+    </p>
+    <div class="tab">
+        <button class="btn_course" type="button" data-bs-toggle="collapse" data-bs-target="#collapse1">
+            Baby Poneys
+        </button>    
+    <div class="collapse" id="collapse1">
+        <div class="card card-body">
+            Âge minimum: à partir de 18 mois </br>
+        <div class="img_course">
+            <img src="" alt="Baby Poney photo">
+        </div>
+            Horaires: </br>
+            Tarif: </br>
+        </div>
+    </div> 
+        <button class="btn_course" type="button" data-bs-toggle="collapse" data-bs-target="#collapse2">
+            Eveil Poneys
+        </button>    
+        <div class="collapse" id="collapse2">
+            <div class="card card-body">
+                Âge: 3 à 5 ans </br>
+                <div class="img_course">
+                    <img src="" alt="Eveil Poney photo">
+                </div>
+                Horaires: </br>
+                Tarif: </br>
+            </div>
+        </div>
+        <button class="btn_course" type="button" data-bs-toggle="collapse" data-bs-target="#collapse3">
+            Poneys
+        </button>
+        <div class="collapse" id="collapse3">
+            <div class="card card-body">
+                Âge minimum: 6 ans </br>
+                <div class="img_course">
+                    <img src="" alt="Poney photo">
+                </div>
+                Horaires: </br>
+                Tarif: </br>
+            </div>
+        </div>
+        <button class="btn_course" type="button" data-bs-toggle="collapse" data-bs-target="#collapse4">
+            Cheval
+        </button>
+        <div class="collapse" id="collapse4">
+            <div class="card card-body">
+                Niveau minimum: Galop 3 </br>
+                <div class="img_course">
+                    <img src="" alt="Cheval Privilège photo">
+                </div>
+                Horaires: </br>
+                Tarif: </br>
```

Comment:
why br here?

Reply:
Probably br is not a good practice ?? I removed them to place my text in p tags, the goal is to seperate horaires and tarifs